tslint.json

```diff
@@ -70,6 +70,24 @@
             "arrow-parameter",
             "property-declaration",
             "variable-declaration"
-        ]
+        ],
+        "no-magic-numbers": [
+            true
+        ],
+        "no-reference": true,
+        "curly": true,
```

Comment:
I think `"ignore-same-line"` would make
```javascript
if (/*condition*/) return foo;
```
a valid statement.  I am against that because sometimes you eventually want to add other things to the block and it can cause bugs. what do you think?